src/summary/level-two-routes.component.tsx

```diff
@@ -15,7 +15,7 @@ export function LevelTwoRoutes(props: LevelTwoRoutesProps) {
   });
 
   return (
-    <main className="omrs-main-content">
+    <>
```

Comment:
Minor thing, but this fragment can be removed:

```jsx
return currentPatient && (
  <Switch>
  </Switch>
)
```

Other than that, though, it looks good.